packages/loader/container-loader/src/container.ts

```diff
@@ -1046,9 +1091,13 @@ export class Container
 						// starting to attach the container to storage.
 						// Also, this should only be fired in detached container.
 						this._attachState = AttachState.Attaching;
-						this.context.notifyAttaching(
-							getSnapshotTreeFromSerializedContainer(summary),
-						);
+						this.emit("attaching");
+						const snapshot = getSnapshotTreeFromSerializedContainer(summary);
```

Comment:
this line should move inside the if, as we don't need to do it unless offlineLoadEnabled, as the var isn't used otherwise